src/test/kani/FileNameWithSpace/my src/hi.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit: The path can be simpler (`src/test/kani/FileNameWithSpace/hello there.rs` or similar)

Reply:
I tried that, but it doesn't actually work. Rust doesn't allow space in their crate name. I get the following error even while running stable `rustc`:
```sh
$ rustc "hello there.rs" 
error: invalid character ` ` in crate name: `hello there`

error: aborting due to previous error

```